fe-nc-news-react/src/components/CommentCard.jsx

```diff
@@ -38,8 +62,20 @@ export default function CommentCard({ comment }) {
             <p className="comment-card-info">{formattedDate}</p>
             <p className="comment-card-info">Votes: {comment.votes}</p>
             <p className="comment-card-info">{comment.body}</p>
+            {commentIdDeleted === comment.comment_id ? (
+              <button>Deleted</button>
+            ) : (
+              <button
```

Comment:
we should only show the button to for the comments the user is allowed to delete here